ansible/group_vars/all.yml

```diff
@@ -135,10 +168,12 @@ supervisor_http_password: "{{ secrets.SUPERVISOR_HTTP_PASSWORD | default(None) }
 supervisor_http_username: "{{ secrets.SUPERVISOR_HTTP_USERNAME | default(None) }}"
 # To enable Local ES snapshot override with repository location
 es_local_repo: false
+
 blobdb_snapshot_bucket: dimagi-{{ deploy_env }}-blobdb-backups
 couchdb_snapshot_bucket: dimagi-{{ deploy_env }}-couch-backups
 postgres_snapshot_bucket: dimagi-{{ deploy_env }}-postgres-backups
 es_snapshot_bucket: "dimagi-{{ deploy_env }}-es-snapshots"
-aws_region: None
-# this reads "'s3.{{ aws_region }}.amazonaws.com' if aws_region else None"
-aws_endpoint: '{{ aws_region and "s3." + aws_region + ".amazonaws.com" }}'
+aws_endpoint: "s3.{{ aws_region }}.amazonaws.com"
+
+airflow_secret_key: '{{ secrets.AIRFLOW_SECRET_KEY | default(None) }}'
```

Comment:
I left this like that because it is how all the other secrets are done in this file. Happy to change to something else if that makes sense to you. I think having this as the string 'None' will break airflow since it expects a fernet key, which probably makes this a safe thing to do.